penny_university_frontend/src/config.ts

```diff
@@ -0,0 +1,5 @@
+const config = {
+  sentry: 'https://cfb89b4197334bfebcedeb8c44ffbf06:978d276675c448d2b9f2e1257584cbaf@o410294.ingest.sentry.io/5284085',
```

Comment:
Does this mean that we're storing this in the clear in the javascript in people's browsers? Practically speaking, at this point it isn't going to matter much, but it would be nice to not store it in the clear if we can avoid it.

But maybe I'm wrong. `import` can't be used outside of a module, so I don't know how I would get this value from the console.

Reply:
We could mangle the bundle so that it is harder to get. We could also make the key more secure by including it as a secret in the build process.

The worst that happens is that someone starts to send spam errors.